lib/internal/Magento/Framework/File/Pdf/Image.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\File\Pdf;
+
+use Magento\Framework\File\Pdf\ImageResource\ImageFactory;
+use Zend_Pdf_Image;
+
+class Image extends Zend_Pdf_Image
```

Comment:
Can we use the `Zend_Pdf_Image` class in the constructor instead of extending it and replicating all methods of the `Zend_Pdf_Image` here in this class?

Reply:
I think yes. I can completely remove Zend_Pdf_Image from  lib/internal/Magento/Framework/File/Pdf/Image.php

@engcom-Hotel I'll remove that extending